XenModel/Network/NetworkingHelper.cs

```diff
@@ -40,20 +40,20 @@ namespace XenAdmin.Network
     public class NetworkingHelper
     {
         /// <summary>
-        /// Returns the bond that the master is using as its management interface, or null if
+        /// Returns the bond that the coordinator is using as its management interface, or null if
         /// such a thing cannot be found.
         /// </summary>
         /// <param name="conn"></param>
         /// <returns></returns>
-        public static Bond GetMasterManagementBond(IXenConnection conn)
+        public static Bond GetCoordinatorManagementBond(IXenConnection conn)
         {
-            PIF pif = GetMasterManagementPIF(conn);
+            PIF pif = GetCoordinatorManagementPIF(conn);
             return pif == null ? null : pif.BondMasterOf();

```

Comment:
BondMasterOf

Reply:
Excluded as it's part of XenAPI.

Reply:
BondMasterOf() is also an API extension and can change.